lib/vtls/sectransp.c

```diff
@@ -1090,9 +1090,9 @@ static OSStatus CopyIdentityWithLabel(char *label,
 #if CURL_BUILD_IOS
           common_name = SecCertificateCopySubjectSummary(cert);
 #elif CURL_BUILD_MAC_10_7
-          SecCertificateCopyCommonName(cert, &common_name);
+          OSStatus copy_status = SecCertificateCopyCommonName(cert, &common_name);
 #endif
-          if(CFStringCompare(common_name, label_cf, 0) == kCFCompareEqualTo) {
+          if(copy_status == errSecSuccess && CFStringCompare(common_name, label_cf, 0) == kCFCompareEqualTo) {
```

Comment:
`copy_status` is used here in the generic code while declared for macOS >= 10.7 only. The declaration needs to be moved, right?

Reply:
You are right. i was too fast. Fixing.

Reply:
Done.